deploy/charts/cert-manager/values.yaml

```diff
@@ -91,6 +91,16 @@ securityContext: {}
 #   runAsUser: 1000
 #   runAsNonRoot: true
 
+# Container Security Context
+# ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+containerSecurityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
```

Comment:
Hm, should we be providing our own default securityContext that has strong default configuration? 🤔

Reply:
I think that's a great idea. I haven't tested with a configuration in place yet, but I can next week.

Reply:
Thinking about this further, it's going to probably need more testing to make sure it works by default. Perhaps a better idea for a future PR.